Update all `<feature>:<action>` references with `<feature>:<permission>` instead for clarity and add examples (#2284)
clerkinc/clerk-docs: docs/organizations/create-roles-permissions.mdx

## Create a new permission for your organization

1. In the Clerk Dashboard, navigate to [**Permissions**](https://dashboard.clerk.com/last-active?path=organizations-settings/permissions).

1. Select **Create new permission**.

1. Give the permission a name, a key to reference it by in the format `org:<feature>:<permission>`, and a description.
   > [!NOTE]
   > Common permission values could be:
   >
   > - `create` — to allow creating resources
   > - `read` — to allow reading/viewing resources
   > - `update/manage` — to allow updating/editing resources
   > - `delete` — to allow deleting resources
   >

>    > For example, you could create a new role called **Sales** (`org:sales`) and a new permission called **Create invoices** (`org:invoices:create`) which allows only users with this permission to edit invoices. You could also grant this permission to the **Billing** role.

Reviewer: ```suggestion
   > For example, you could create a new permission called **Create invoices** (`org:invoices:create`) which allows only users with this permission to edit invoices. Then, you could assign this permission to a role, or multiple roles, such as **Billing** (`org:billing`) or **Sales** (`org:sales`).
```

Reviewer: yeah let's do it ! :) great catch
